scripts/config/msu/utils.nut

```diff
@@ -60,12 +61,16 @@
 
 				case "array":
 					_out.writeU8(this.DataType.Array);
-					::MSU.serialize(element, _out);
+					this.serialize(element, _out);
```

Comment:
Split the changing of `::MSU.` to `this` into a separate `refactor` commit to keep it separate from the `feat` of adding the Null data type.

Reply:
I'll just put that as a fix on dev